ios/FluentUI/Pill Button Bar/PillButton.swift

```diff
@@ -97,7 +105,7 @@ open class PillButton: UIButton {
 
     private func setupView() {
         setTitle(pillBarItem.title, for: .normal)
-        titleLabel?.font = Constants.font
+        titleLabel?.font = style == .transparent ? Fonts.button1 : Constants.font
```

Comment:
> titleLabel?.font [](start = 7, length = 17)

I think it'd be cleaner to just call setupFont here

Reply:
I think, in this case, a better approach is linked to @Saadnajmi comment but a bit different, to move Fonts.button1 to Constants as it is used in more than one place, and just because it is 1 line of code, do you think it needs an extra method?

Reply:
I think it's better to not duplicate this code, even if it's 1 line. This will make sure that a future potential change in  this configuration is addressed in all cases, reducing the likelihood of bugs.